ledger/artifactmanager/middleware.go

```diff
@@ -67,6 +67,21 @@ func newMiddleware(
 	}
 }
 
+// In Build it should follow after checkJet, since it expects jet key
+func (m *middleware) addFieldsToLogger(handler core.MessageHandler) core.MessageHandler {
+	return func(ctx context.Context, parcel core.Parcel) (core.Reply, error) {
+		context, _ := inslogger.WithField(ctx, "targetid", parcel.DefaultTarget().String())
```

Comment:
this whole change to ledger's handlers is wrong. addFieldsToLogger applied after checkJet wrapper, so "targetid" field is not applied to checkJet. Second part that takes jet id from context after checkJet should be moved to checkJet code as that code fetches and provides jet id for the rest of the code.

Reply:
fixed